osu.Game/Screens/Edit/Editor.cs

```diff
@@ -80,7 +80,7 @@ public partial class Editor : ScreenWithBeatmapBackground, IKeyBindingHandler<Gl
 
         public override float BackgroundParallaxAmount => 0.1f;
 
-        public override bool AllowBackButton => false;
+        public override bool AllowUserExit => false;
```

Comment:
Am I the only one weirded out by this sort of thing? Why does editor apparently "not allow user exit"? The user can exit it just fine.

Reply:
Looks to have been done this way historically so the back button doesn't display, while still allowing the user to exit via `Back` action by manual handling in `Editor.OnPressed`.

This can make better sense with this PR by setting the initial back button state to hidden instead of overriding this.

Reply:
This should hopefully not cause any behavioural change to the editor due to the handling of the exit action moving from `Editor.OnPressed` all the way back to `BackReceptor`.